requirements/dev.txt

```diff
@@ -5,92 +5,85 @@
 #    make upgrade
 #
 -e git+https://github.com/edx/acid-block.git@98aecba94ecbfa934e2d00262741c0ea9f557fc9#egg=acid-xblock
-appdirs==1.4.3            # via fs
-astroid==1.6.6            # via pylint, pylint-celery
-atomicwrites==1.3.0       # via pytest
+appdirs==1.4.3            # via fs, virtualenv
+astroid==2.3.3            # via pylint, pylint-celery
 attrs==19.3.0             # via pytest
-backports.functools-lru-cache==1.6.1  # via astroid, isort, pylint
-backports.os==0.1.1       # via fs
 binaryornot==0.4.4        # via cookiecutter
 bok_choy==0.7.1
-boto3==1.4.8              # via fs-s3fs
+boto3==1.12.5             # via fs-s3fs
 boto==2.39.0
-botocore==1.8.50          # via boto3, s3transfer
+botocore==1.15.5          # via boto3, s3transfer
 certifi==2019.11.28       # via requests
 chardet==3.0.4            # via binaryornot, requests
 click-log==0.3.2          # via edx-lint
 click==7.0                # via click-log, edx-lint
-configparser==4.0.2       # via importlib-metadata, pydocstyle, pylint
-contextlib2==0.6.0.post1  # via importlib-metadata
 cookiecutter==0.9.0
-coverage==4.5.4
+coverage==5.0.3
 ddt==1.2.2
+distlib==0.3.0            # via virtualenv
 django-pyfs==2.0
-django==1.11.26
+django==2.2.10
 docutils==0.15.2          # via botocore
 edx-lint==1.4.1
-enum34==1.1.6             # via astroid, fs
-filelock==3.0.12          # via tox
-fs-s3fs==0.1.5
+filelock==3.0.12          # via tox, virtualenv
+fs-s3fs==0.1.8
 fs==2.0.27                # via django-pyfs, fs-s3fs, xblock
-funcsigs==1.0.2           # via mock, pytest
-future==0.18.2            # via backports.os
-futures==3.3.0 ; python_version == "2.7"  # via isort, s3transfer
 idna==2.8                 # via requests
-importlib-metadata==1.3.0  # via pluggy, pytest, tox
+importlib-metadata==1.5.0  # via pluggy, pytest, tox, virtualenv
+importlib-resources==1.0.2  # via virtualenv
 isort==4.3.21
-jinja2==2.10.3            # via cookiecutter
+jinja2==2.11.1            # via cookiecutter
 jmespath==0.9.4           # via boto3, botocore
 lazy-object-proxy==1.4.3  # via astroid
 lazy==1.1
 lxml==3.8.0
-mako==1.1.0
+mako==1.1.1
 markupsafe==1.1.1         # via jinja2, mako, xblock
 mccabe==0.6.1             # via pylint
 mock==3.0.5
-more-itertools==5.0.0     # via pytest, zipp
+more-itertools==8.2.0     # via pytest
```

Comment:
@kaizoku, this [breaks](https://travis-ci.org/edx/xblock-sdk/jobs/654252324#L475) the Python 2.7 (which I think it still supported, based on the documentation in the README file) integration tests.